src/main/java/uk/gov/hmcts/darts/automation/utils/JsonApi.java

```diff
@@ -0,0 +1,282 @@
+package uk.gov.hmcts.darts.automation.utils;
+
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.filter.log.LogDetail;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import io.restassured.http.Headers;
+import io.restassured.http.Cookie;
+import io.restassured.http.Cookies;
+import io.restassured.response.*;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+
+import static io.restassured.RestAssured.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.Duration;
+import java.util.List;
+
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import org.junit.jupiter.api.Test;
+
+
+
+public class JsonApi {
+	private static Logger log = LogManager.getLogger("JsonApi");
+    static Response response;
+    String authorization;
+	static String baseUri = ReadProperties.main("jsonApiUri");
+
+
+	public JsonApi() {
+		
+	}
+
+    public static RequestSpecification requestLogLevel(LogDetail loggingLevel){
+        RequestSpecification requestSpec = new RequestSpecBuilder().log(loggingLevel).build();
+        return requestSpec;
+    }
+
+    public static ResponseSpecification responseLogLevel(LogDetail loggingLevel){
+        ResponseSpecification loglevel = new ResponseSpecBuilder().log(loggingLevel).build();
+        return loglevel;
+    }
+    
+    public String authenticate() {
+    	log.info("authentication");
+    	response  = 
+    		given()
+    			.spec(requestLogLevel(ReadProperties.authRequestLogLevel))
+    			.header("Content-Type","application/x-www-form-urlencoded")  
```

Comment:
Should these strings all be constants as they are repeated throughout?

Reply:
I am refactoring this class as part of DMP-847 & will do that there